client/layout/style.scss

```diff
@@ -34,17 +34,12 @@
 	}
 
 	// Themes sets it own padding/margin
-	.is-section-theme.has-no-sidebar &,
+	.is-section-theme &,
 	.is-section-themes.has-no-sidebar & {
 		padding: 0;
 		margin: 0;
 	}
 
-	.is-section-theme & {
```

Comment:
I think there's another set of very similar styling on lines 54 -> 62 that I had introduced in the pull request.

I can pull these out after the demo today if you don't mind waiting for that.

Reply:
Yes, np. I just reverted the first commit you added. Feel free to add/remove what is needed :)